src/scheduler/batch/__tests__/helpers.test.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  DateContext,
+  getDateContexts,
+  schedulerMilestoneDays
+} from "../helpers";
+
+function testContext(actualUnpublishContext: DateContext[], expectedDateContexts: DateContext[], daysToAction: number) {
+  const actualContext = actualUnpublishContext.find(
+    (context) => context.eventMilestone === daysToAction
+  );
+  const expectedContext = expectedDateContexts.find(
+    (context) => context.eventMilestone === daysToAction
+  );
+  expect(actualContext?.eventMilestone).toBe(expectedContext?.eventMilestone);
+  expect(actualContext?.eventDate.toString()).toBe(expectedContext?.eventDate.toString());
+}
+
+describe("Date Contexts", () => {
+  describe("annual review date contexts", () => {
+    const expectedDateContexts: DateContext[] = [
+      {
+        eventDate: new Date(
+          Date.UTC(2022, 11, 5, 0, 0, 0)
```

Comment:
you should be able to get away with just `new Date(2022, 11, 5, 0, 0, 0)`.. tbh I'm not entirely sure why Date.UTC is a thing. minor, anyway.

Reply:
new Date() will include the local timezone of the system its running on whereas Date.UTC will default to UTC timezone regardless of the system settings.  Shouldn't really matter either way except if you're running the tests at between 11pm - 12am during daylight savings.

Reply:
It should be that if you use 3+ parameters rather than something like "10/11/12" it should be UTC but we can leave for now but no harm in being doubly sure